windowpopup.js

```diff
@@ -0,0 +1,249 @@
+const projects = [
+  {
+    title: 'Tonic',
+    roles: { company: 'Canopy', position: ['<span class="fw-bold">Canopy</span>', 'Back End Dev', '2015'] },
+    image: 'img/Desktop/Snapshoot-Portfolio1.svg',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    skills: ['html', 'Ruby on rails', 'css', 'javascript'],
+    skillsModal: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstraps'],
+    livelink: { link: 'felix45.github.io/portfolio', text: 'See live' },
+    seesource: { link: 'github.com/felix45/portfolio', text: 'See Source' },
+    seeproject: 'See Project',
+    htmlClass: { article: ['card', 'article-card', 'mb-3', 'm-sm-auto', 'p-sm-2', 'hide-ruby'], cardImage: ['col-md-6', 'images'], cardDescription: ['col-md-6'] },
+    rowClass: { row: ['row', 'g-0', 'p-2'] },
+  },
+
```

Comment:
- [ ] Please, the popup must be responsive, it fits mobile and desktop sizes properly.

Reply:
> As suggested by the previous reviewer , kindly make sure that the popup is responsive and fits properly to mobile and desktop sizes. For this you have to do the following:-
> 
> Mobile Popup:
> 
> * [ ]  Kindly make sure that the popup is of the same height and width as per the Figma design. As you can see that your popup heading is not visible. Do fix it.
> * [ ]  Optional: Kindly make sure that the background of the popup is blurred as per the Figma design.
> * [ ]  Optional: Kindly make sure to follow the font size for the paragraph as per the given design.
> * [ ]  Kindly follow the colors, spacing, and font for the entire popup and replicate as much as possible.

Please note that you still need to work on this as suggested by the previous reviewer, to make your page UI (user interface) better and to match the original Figma design💪🏽👍

Reply:
@rotimiazeez Kindly help me understand what you mean by doesn't fit screen. It is displaying perfectly on my side.The popup on your portfolio too does not fit on my screen, I don't get why you're all flagging a 'mistake' yourselves are unable to implement.

Reply:
@Mugiira-Kim Kindly check the requirement and original design to make sure your design is a match.😉
Your design | Original design
![screenshot (46)](https://user-images.githubusercontent.com/44624138/168055365-fbc6831b-5146-4788-b6c9-e457bf322d74.png) | ![screenshot (48)](https://user-images.githubusercontent.com/44624138/168060061-4c615f1c-2753-4eed-909d-0ae18533dc5f.png).

Note that you can click on each element to see the style properties e.g font-family, font-size, color, width, height ... on the right side of your screen as seen in the screenshot.👍